docs/fast_data/bucket_storage_support/configuration/ingestion_reloader.md

```diff
@@ -198,20 +205,96 @@ paths:
 The Ingestion Reloader exposes four `POST` routes to trigger its functionalities.
 
 ### Reingestion of a Topic
-
-This route allows the user to reingest a topic partially or from the beginning.
-The user has to specify the "stored" topic name and the "reingestion" topic name, optionally a filter on the timestamp of the topic may also be specified (with the `startDate` and `stopDate` properties). In this last case, only the records included in the specified time interval will be reingested on the selected topic.
+This route allows the user to reingest a topic partially or from the beginning. The user has to specify the "stored" topic name and the "reingestion" topic name. Optionally, a filter on the timestamp of the topic may also be specified (with the startDate and stopDate properties). In this last case, only the records included in the specified time interval will be reingested on the selected topic. Additionally, the user can specify a customSelectorName to use a custom selector script for filtering the messages to be reingested.
 
 ### Reingestion of a File
-
-This route allows the user to reingest a single file from the bucket into the topic specified in the request.
-The user has to specify the file name and the reingestion topic.
+This route allows the user to reingest a single file from the bucket into the topic specified in the request. The user has to specify the file name and the reingestion topic. Additionally, the user can specify a customSelectorName to use a custom selector script for filtering the messages to be reingested.
 
 ### Stopping reingestion
-
-This route allows the user to stop the ongoing reingestion of a topic.
-The user has to specify the _ingestion_ topic that is currently being reingested.
+This route allows the user to stop the ongoing reingestion of a topic. The user has to specify the ingestion topic that is currently being reingested.
 
 :::caution
 This route can stop only the reingestion of a topic, not the one of a single file.
 :::
+
+Please refer to the [Custom Selector Feature section](#customer-selectors) for more details on how to use the custom selector feature for precise and targeted filtering of messages to be reingested.
+## Customer Selectors
+
+The customSelector feature is a powerful addition to the Ingestion Reloader service that allows users to apply custom filtering logic to select specific messages from the bucket for re-ingestion into the Kafka topic.
```

Comment:
I would provide a quick example here or a brief explanation of a possible use case, to let the user better  understand the feature

Reply:
Maybe it is possible to provide a cURL as an example?